week1/react-exercises/src/2-guarantee/Guarantee.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const Guarantee = () => {
+  const guarantees = [
+    {
+      img: "./images/f-delivery.png",
+      title: "Free shipping",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+    {
+      img: "./images/coin.png",
+      title: "100% Money back",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+    {
+      img: "./images/chat.png",
+      title: "Online support 24/7",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+  ];
+
+  return (
+    <div className="gContainer">
+      {guarantees.map((guar, i) => (
+        <GuaranteeLayout
+          key={i}
+          img={guar.img}
+          title={guar.title}
+          desc={guar.description}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
- You can just call it `container` if it is component specific or `guarantees-container` if you want to be explicit
- Don't use the index for a key
- Try not to use abbreviations it makes the code less readable 

```suggestion
  return (
    <div className="container">
      {guarantees.map((guarantee, i) => (
        <GuaranteeLayout
          key={guarantee.title}
          img={guarantee.img}
          title={guarantee.title}
          desc={guarantee.description}
        />
      ))}
    </div>
  );
};
```